### ISSUE TITLE
SmartCity-Accessibility Server Side Unit Tests #814 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#414]
Build 'SmartCity-Accessibility Server Side Unit Tests' is failing!

Last 50 lines of build :
```
Started by GitHub push by KaplanAlexander
Building in workspace /var/lib/jenkins/sharedspace/SmartCity-Accessibility
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/TechnionYP5777/SmartCity-Accessibility.git # timeout=10
Fetching upstream changes from https://github.com/TechnionYP5777/SmartCity-Accessibility.git
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/TechnionYP5777/SmartCity-Accessibility.git +refs/heads/*:refs/remotes/origin/*
 > git rev-parse refs/remotes/origin/master^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/master^{commit} # timeout=10
Checking out Revision df1b10ddea3b9cb6a1e1840e58c33f2f6b2ae044 (refs/remotes/origin/master)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f df1b10ddea3b9cb6a1e1840e58c33f2f6b2ae044
 > git rev-list 990cade6052834ca3e57f4abcc49b6436e2fe0db # timeout=10
Injecting SonarQube environment variables using the configuration: Sonar
[SmartCity-Accessibility] $ /var/lib/jenkins/tools/hudson.tasks.Maven_MavenInstallation/Maven/bin/mvn -f server-side/pom.xml clean -Dram=1024 org.jacoco:jacoco-maven-plugin:prepare-agent test -Dsonar.coverage.exclusions=**gui** -Dgroups=smartcity.accessibility.categories.UnitTests
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [Alexander Kaplan] df1b10ddea3b9cb6a1e1840e58c33f2f6b2ae044 - maven dependencies plugin #413

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Unit%20Tests/814/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility Server Side Unit Tests' is failing!

Last 50 lines of build :
```
Started by GitHub push by KaplanAlexander
Building in workspace /var/lib/jenkins/sharedspace/SmartCity-Accessibility
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/TechnionYP5777/SmartCity-Accessibility.git # timeout=10
Fetching upstream changes from https://github.com/TechnionYP5777/SmartCity-Accessibility.git
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/TechnionYP5777/SmartCity-Accessibility.git +refs/heads/*:refs/remotes/origin/*
 > git rev-parse refs/remotes/origin/master^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/master^{commit} # timeout=10
Checking out Revision df2d8e8c35c0ae0f7991806dda4ff206e9ec35fc (refs/remotes/origin/master)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f df2d8e8c35c0ae0f7991806dda4ff206e9ec35fc
 > git rev-list df1b10ddea3b9cb6a1e1840e58c33f2f6b2ae044 # timeout=10
Injecting SonarQube environment variables using the configuration: Sonar
[SmartCity-Accessibility] $ /var/lib/jenkins/tools/hudson.tasks.Maven_MavenInstallation/Maven/bin/mvn -f server-side/pom.xml clean -Dram=1024 org.jacoco:jacoco-maven-plugin:prepare-agent test -Dsonar.coverage.exclusions=**gui** -Dgroups=smartcity.accessibility.categories.UnitTests
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [Alexander Kaplan] df1b10ddea3b9cb6a1e1840e58c33f2f6b2ae044 - maven dependencies plugin #413

- [Alexander Kaplan] df2d8e8c35c0ae0f7991806dda4ff206e9ec35fc - procfile for heroku #413



[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Unit%20Tests/815/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility Server Side Unit Tests' is failing!

Last 50 lines of build :
```
Started by GitHub push by KaplanAlexander
Building in workspace /var/lib/jenkins/sharedspace/SmartCity-Accessibility
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/TechnionYP5777/SmartCity-Accessibility.git # timeout=10
Fetching upstream changes from https://github.com/TechnionYP5777/SmartCity-Accessibility.git
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/TechnionYP5777/SmartCity-Accessibility.git +refs/heads/*:refs/remotes/origin/*
 > git rev-parse refs/remotes/origin/master^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/master^{commit} # timeout=10
Checking out Revision 12bb806574e886ab57feb2dd084313977f4a012c (refs/remotes/origin/master)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 12bb806574e886ab57feb2dd084313977f4a012c
 > git rev-list df2d8e8c35c0ae0f7991806dda4ff206e9ec35fc # timeout=10
Injecting SonarQube environment variables using the configuration: Sonar
[SmartCity-Accessibility] $ /var/lib/jenkins/tools/hudson.tasks.Maven_MavenInstallation/Maven/bin/mvn -f server-side/pom.xml clean -Dram=1024 org.jacoco:jacoco-maven-plugin:prepare-agent test -Dsonar.coverage.exclusions=**gui** -Dgroups=smartcity.accessibility.categories.UnitTests
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building SmartCity-Accessibility 0.5.1-SNAPSHOT
[INFO] ------------------------------------------------------------------------
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [Alexander Kaplan] df1b10ddea3b9cb6a1e1840e58c33f2f6b2ae044 - maven dependencies plugin #413

- [Alexander Kaplan] df2d8e8c35c0ae0f7991806dda4ff206e9ec35fc - procfile for heroku #413

- [Alexander Kaplan] 12bb806574e886ab57feb2dd084313977f4a012c - slashes are confusing #413



[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Unit%20Tests/816/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility Server Side Unit Tests' is failing!

Last 50 lines of build :
```
Started by GitHub push by KaplanAlexander
Building in workspace /var/lib/jenkins/sharedspace/SmartCity-Accessibility
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/TechnionYP5777/SmartCity-Accessibility.git # timeout=10
Fetching upstream changes from https://github.com/TechnionYP5777/SmartCity-Accessibility.git
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/TechnionYP5777/SmartCity-Accessibility.git +refs/heads/*:refs/remotes/origin/*
 > git rev-parse refs/remotes/origin/master^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/master^{commit} # timeout=10
Checking out Revision 14633708da692a7e4b1f874fdac5729623c4f376 (refs/remotes/origin/master)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 14633708da692a7e4b1f874fdac5729623c4f376
 > git rev-list 12bb806574e886ab57feb2dd084313977f4a012c # timeout=10
Injecting SonarQube environment variables using the configuration: Sonar
[SmartCity-Accessibility] $ /var/lib/jenkins/tools/hudson.tasks.Maven_MavenInstallation/Maven/bin/mvn -f server-side/pom.xml clean -Dram=1024 org.jacoco:jacoco-maven-plugin:prepare-agent test -Dsonar.coverage.exclusions=**gui** -Dgroups=smartcity.accessibility.categories.UnitTests
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building SmartCity-Accessibility 0.5.1-SNAPSHOT
[INFO] ------------------------------------------------------------------------
Downloading: https://jitpack.io/com/nimbusds/lang-tag/maven-metadata.xml
Downloading: https://jitpack.io/com/nimbusds/nimbus-jose-jwt/maven-metadata.xml
Downloading: https://jitpack.io/net/minidev/json-smart/maven-metadata.xml
[INFO] 
[INFO] --- maven-clean-plugin:2.6.1:clean (default-clean) @ SmartCity-Accessibility ---
[INFO] Deleting /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target
[INFO] 
[INFO] --- jacoco-maven-plugin:0.7.8:prepare-agent (default-cli) @ SmartCity-Accessibility ---
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [Alexander Kaplan] df1b10ddea3b9cb6a1e1840e58c33f2f6b2ae044 - maven dependencies plugin #413

- [Alexander Kaplan] df2d8e8c35c0ae0f7991806dda4ff206e9ec35fc - procfile for heroku #413

- [Alexander Kaplan] 12bb806574e886ab57feb2dd084313977f4a012c - slashes are confusing #413

- [Alexander Kaplan] 14633708da692a7e4b1f874fdac5729623c4f376 - remove stdout print debug



[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Unit%20Tests/817/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
